x-pack/plugins/security_solution/common/search_strategy/security_solution/hosts/common/index.ts

```diff
@@ -126,3 +127,21 @@ export interface HostHit extends Hit {
 }
 
 export type HostHits = Hits<number, HostHit>;
+
+export const enum HostRiskScoreFields {
+  timestamp = '@timestamp',
+  hostName = 'host.name',
+  riskScore = 'risk_stats.risk_score',
+  risk = 'risk',
+  // TODO: Steph/Host Risk
```

Comment:
Might help to link TO-Dos to created tickets so they dont get forgotten

Reply:
im deleting them in my branch, lets get this merged asap